src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.cpp

```diff
@@ -441,7 +440,6 @@ void SmokeCoAlarmServer::HandleRemoteSelfTestRequest(CommandHandler * commandObj
         return;
     }
 
-    VerifyOrReturn(SetTestInProgress(endpointId, true), commandObj->AddStatus(commandPath, Status::Failure));
```

Comment:
This change is not clearly explained in the commit message.  Please move it out into a separate PR, with a _very_ clear commit message explanation for why this change is being made.  Because I don't understand why we're pushing this work onto the SDK consumer.

Reply:
This part of the change has been removed.